src/subscribe/tests.rs

```diff
@@ -16,14 +16,14 @@ use super::{Client, Conn};
 use crate::{
     client::Certs,
     model::{Interval, Period, Policy, TimeSeries},
-    to_cert_chain, to_private_key, to_root_cert,
+    to_ca_certs, to_cert_chain, to_private_key,
 };
 
 pub(crate) static TOKEN: LazyLock<Mutex<u32>> = LazyLock::new(|| Mutex::new(0));
 
 const CERT_PATH: &str = "tests/cert.pem";
 const KEY_PATH: &str = "tests/key.pem";
-const CA_CERT_PATH: &str = "tests/root.pem";
+const CA_CERT_PATH: &str = "tests/ca_certs.pem";
```

Comment:
```suggestion
const CA_CERT_PATH: &str = "tests/ca_cert.pem";
```

This is a suggestion in line with the previous comment.

Reply:
I modified the path name.